### PR TITLE
Update ddp tutorial to tell user that TcpStore has been enabled on Windows platform

### PR DESCRIPTION
**Pull request by gunandrose4u**

### CODE REVIEW

**Comment by gunandrose4u** [comment]
@mrshenli , please help to review ddp_tutorial updates for enable TcpStore on Windows. Thank you!

File: intermediate_source/ddp_tutorial.rst [hunk 1]
```diff
@@ -80,28 +80,20 @@ be found in
 
     from torch.nn.parallel import DistributedDataParallel as DDP
 
+    # Distributed package only covers collective communications with Gloo
+    # backend, FileStore and TcpStore on Windows platform.
```
Comment (mrshenli) [inline intermediate_source/ddp_tutorial.rst:84]: nit: trying to make it more explicit

On Windows platform, the torch.distributed package only supports Gloo backend,
FileStore and TcpStore.

**Review by mrshenli — APPROVED** [review]
LGTM!

**Comment by mrshenli** [comment]
Hey @jlin27, the CI failure seem irrelevant to this PR. Shall we land?

```
Jan 28 06:48:44 Collecting torch (from -r /var/lib/jenkins/workspace/.jenkins/../requirements.txt (line 9))
Jan 28 06:48:44   Downloading https://files.pythonhosted.org/packages/90/4f/acf48b3a18a8f9223c6616647f0a011a5713a985336088d7c76f3a211374/torch-1.7.1-cp36-cp36m-manylinux1_x86_64.whl (776.8MB)
Jan 28 06:48:52 Exception:
Jan 28 06:48:52 Traceback (most recent call last):
Jan 28 06:48:52   File "/opt/conda/lib/python3.6/site-packages/pip/_vendor/urllib3/contrib/pyopenssl.py", line 294, in recv_into
Jan 28 06:48:52     return self.connection.recv_into(*args, **kwargs)
Jan 28 06:48:52   File "/opt/conda/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1822, in recv_into
Jan 28 06:48:52     self._raise_ssl_error(self._ssl, result)
Jan 28 06:48:52   File "/opt/conda/lib/python3.6/site-packages/OpenSSL/SSL.py", line 1639, in _raise_ssl_error
Jan 28 06:48:52     raise SysCallError(errno, errorcode.get(errno))
Jan 28 06:48:52 OpenSSL.SSL.SysCallError: (104, 'ECONNRESET')
Jan 28 06:48:52 
Jan 28 06:48:52 During handling of the above exception, another exception occurred:
```

File: intermediate_source/ddp_tutorial.rst (hunk 1 above)
Comment (gunandrose4u) [reply intermediate_source/ddp_tutorial.rst]: Resolved

**Review by malfet — APPROVED** [review]

**Review by malfet — APPROVED** [review]

**Comment by gunandrose4u** [comment]
Thank you for merging the PR! @malfet, @mrshenli

**Pull request merged**